configs/edison/config.20161117.beta0.A_WCYCL1850.ne30_oEC.edison

```diff
@@ -78,6 +79,27 @@ startYear = 41
 # the last year over which to average climatalogies
 endYear = 50
 
+# The comparison grid is a lat/lon grid with 0.5 degree spacing
+lat = numpy.linspace(-90, 90, 361)
+lon = numpy.linspace(-180, 180, 721)
+latLonUnits = degrees
+
+# The name of the mapping file used for interpolation.  If a mapping
+# file has already been generated, supplying the absolute path can save
+# the time of generating a new one.  If a relative path is supplied, the
+# file will be added to the mapping subdirectory within the output base
+# directory, and will be cached for future analysis runs.
+mpasMappingFile = mpasClimatologyMapping.nc
```

Comment:
since we already have the mapping files for these specific cases, wouldn't we want to add the absolute paths for these config files? I guess that's not important though.